packages/wasmkit/src/lib/deploy/contract.ts

```diff
@@ -304,6 +305,10 @@ export class Contract {
       transferAmount,
       customFees,
       memo
-    ).finally(() => { this.stopLoadingAnimation(); });
+    ).then((result) => {
+      this.stopLoadingAnimation();
+      console.log(`TransactionHash: '${chalk.green(result.transactionHash)}'`);
```

Comment:
This has to handle the fail/success result of transaction result. We only get txnHash if the result was successful and in that case returned response code is `0`, so here you can check the code value, if 0 then print sucess with txnHash, else failure with failure reason and returned code.

You can see what the structure of response is in both cases by running them manually by sending a correct and incorrect txn and logging the full response and see where the code value is in the response.

Also need to handle the color coding of success/failure with chalk

Reply:
Sir, I am having trouble locating the response code which is supposed to be checked